pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -88,7 +88,7 @@ func (enqueue *Action) Execute(ssn *framework.Session) {
 		}
 	}
 
-	klog.V(3).Infof("Try to enqueue PodGroup to %d Queues", len(jobsMap))
+	klog.V(3).Infof("Try to enqueue PodGroup to %d Queues", queues.Len())
```

Comment:
This queues is `PriorityQueue` , not CRD `Queue`. Please reset to original

Reply:
fixed